lib/src/dio-interceptor-wrapper.dart

```diff
@@ -31,10 +31,12 @@ class SuperTokensInterceptorWrapper extends Interceptor {
       ));
       return;
     }
+    _preRequestLocalSessionState =
```

Comment:
Why did we need to move this at all?

Reply:
because on onResponse requires a value for `_preRequestLocalSessionState` but since we return out of this method before this variable is set in case of a request to non superTokens url this causes an `LateInitializationError`